File: ehmp_1.2.0/ehmp_1.2.0/ehmp_1.2.0/ehmp/product/production/hmp-main/src/main/java/gov/va/cpe/order/Schedule.java
package gov.va.cpe.order;

import gov.va.cpe.vpr.pom.AbstractPOMObject;

import java.util.Map;

public class Schedule extends AbstractPOMObject {

    private String name;
    private String internal;
    private String externalValue;
    private String scheduleType;

    public Schedule() {
        super(null);
    }

    public Schedule(Map<String, Object> vals) {
        super(vals);    //To change body of overridden methods use File | Settings | File Templates.
    }

    public String getName() {
        return name;
    }

    public String getExternalValue() {
        return externalValue;
    }

    public String getScheduleType() {
        return scheduleType;
    }

    public String getInternal() {
        return internal;
    }

    @Override
    public String getSummary() {
        return getName();
    }
}

File: ehmp_1.2.0/ehmp_1.2.0/ehmp_1.2.0/ehmp/product/production/hmp-main/src/main/js/gov/va/cpe/order/Schedule.js
Ext.define('gov.va.cpe.order.Schedule', {
	extend: 'Ext.data.Model',
    idProperty:'uid',
    fields: [
        {name: 'name', type: 'string'},
        {name: 'uid', type: 'string'},
        {name:'externalValue', type: 'string'},
        {name:'scheduleType', type: 'string'},
        {name:'summary', type: 'string'},
        {name: 'internal', type: 'string'}
    ]
});